python/tvm/testing/__init__.py

```diff
@@ -16,41 +16,17 @@
 # under the License.
 # pylint: disable=redefined-builtin, wildcard-import
 """Utility Python functions for TVM testing"""
-from .utils import (
-    assert_allclose,
-    assert_prim_expr_equal,
-    check_bool_expr_is_true,
-    check_int_constraints_trans_consistency,
-    check_numerical_grads,
-    device_enabled,
-    device_test,
-    echo,
-    enabled_targets,
-    exclude_targets,
-    fixture,
-    parameter,
-    parameters,
-    parametrize_targets,
-    uses_gpu,
-    known_failing_targets,
-    object_use_count,
-    requires_cuda,
-    requires_cudagraph,
-    requires_gpu,
-    requires_llvm,
-    requires_rocm,
-    requires_rpc,
-    requires_tensorcore,
-    requires_metal,
-    requires_micro,
-    requires_opencl,
-    test_check_eq_callback,
-    test_raise_error_callback,
-    test_wrap_callback,
-    _auto_parametrize_target,
-    _count_num_fixture_uses,
-    _remove_global_fixture_definitions,
-    _parametrize_correlated_parameters,
-)
+from .utils import assert_allclose, assert_prim_expr_equal, check_bool_expr_is_true
```

Comment:
let's keep the original style (merge everything in a single import)
```
from .utils import (
 ...
 ...
)
```
we also need to import everything registered via `TVM_REGISTER_GLOBAL` here https://github.com/apache/tvm/blob/main/src/support/ffi_testing.cc#L152 e.g.
```
from ._ffi_utils import (
  nop, echo, 
  ...
)
```

Reply:
This is the import style I saw in other [__init__.py ](https://github.com/apache/tvm/blob/8302524a381655a727db74db97777cba99a2609e/python/tvm/te/__init__.py#L21). FrontendTestModule is added.